build.gradle

```diff
@@ -277,16 +280,21 @@ dependencies {
 
   implementation group: 'org.springdoc', name: 'springdoc-openapi-ui', version: versions.springDoc
 
-  implementation group: 'com.github.hmcts.java-logging', name: 'logging', version: versions.logging
+  implementation group: 'com.github.hmcts.java-logging', name: 'logging', version: versions.reformLogging
```

Comment:
Can you change this to
implementation group: 'com.github.hmcts', name: 'java-logging', version: versions.reformLogging